examples/baby-llama/baby-llama.cpp

```diff
@@ -0,0 +1,1354 @@
+#include "ggml.h"
+#include <vector>
+#include <assert.h>
+#include <random>
+#include <string.h>
+
+#undef MIN
+#undef MAX
+#define MIN(a, b) ((a) < (b) ? (a) : (b))
+#define MAX(a, b) ((a) > (b) ? (a) : (b))
```

Comment:
Isn't it better to just use `std::min` and `std::max` in C++ rather than this?

Reply:
Ok, I changed the includes and used std::min & std::max instead of the macros.